kitsune/questions/classifiers.py

```diff
@@ -0,0 +1,43 @@
+from langchain.schema.runnable import RunnableLambda, RunnablePassthrough
+
+from kitsune.llm.questions.prompt import spam_parser, spam_prompt, topic_parser, topic_prompt
+from kitsune.llm.utils import get_llm
+from kitsune.products.utils import get_taxonomy
+from kitsune.questions.models import Question
+
+DEFAULT_LLM_MODEL = "gemini-2.5-flash-preview-04-17"
+
+
+def analyze_question(question: Question) -> tuple[bool, dict]:
```

Comment:
If this function is going to also mark the question as spam or set its topic, I think we should change the name to something like `classify_question` instead.